techdocs/script/cli.py

```diff
@@ -22,23 +29,24 @@
 
     if args.command == "copy":
         try:
-            # zz- prefix to make it the last file in the directory listing for example during PR review
-            with IndexLoader.loaded(
-                os.path.join(args.to_path, "zz-index"), fs, not args.dry_run
+            with FileIndexLoader.loaded(
+                os.path.join(args.to_path, INDEX_DIRECTORY), fs, not args.dry_run
             ) as index:
                 Copier(
-                    DeleteDetector(
-                        args.repo,
-                        index,
-                        args.from_path,
-                        args.to_path,
-                        CopyDetector(
+                    UnnecessaryOperationsFilteringDetector(
```

Comment:
Yeah, [about that](https://github.com/PiwikPRO/actions/pull/62#discussion_r1388060243) :D